site/kubernetes/operator/install-topology-operator.md

```diff
@@ -58,6 +58,32 @@ data:
   ca.crt: # ca cert that can be used to validate the webhook's server certificate
   tls.crt: # generated certificate
   tls.key: # generated key</pre>
+3. Add webhook ca certificate in downloaded release manifest `messaging-topology-operator.yaml`. There are 10 admission webhooks, one for each CRD type.
```

Comment:
If there are 10 of them, should this say "CA certificates"?

Reply:
What I was trying to express is add the same generated certificate to 10 different places. I can certainly word it differently to avoid confusion here.

Reply:
Then should it say "Add the web hook CA certificate found in the downloaded release manifest, `messaging-topology-operator.yaml`"?

Reply:
I think the `clientConfig` hint here is good -- v0.8.0 also included CRD conversion webhooks in addition to the validating admission webhooks. (It seems like the conversion webhooks were removed in later versions).